src/github.ts

```diff
@@ -226,53 +227,59 @@ export class GitHub {
     maxFilesChanged = 64,
     retries = 0
   ): Promise<CommitsResponse> {
+    const baseBranch = await this.getDefaultBranch();
```

Comment:
Should this be a method argument where the caller can provide the base branch name?